### PR TITLE
	modified:   README.md

### PR DESCRIPTION
**Pull request by azim254**
what this PR does fixed some files using Babel 7.x from 8 

modify READme.Md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,19 @@
 
 > official website for opensource254
 
+## Reguirements for Debian
```
Comment (stannlee) [inline README.md:6]: It is fine though this is not the purpose of this readme.
This README.MD Assumes that you have node and npm in your system

Comment (azim254) [reply README.md:6]: the reason ni node ya kitambo inaleta errors

Comment (stannlee) [reply README.md:6]: > the reason ni node ya kitambo inaleta errors

Yes, maybe we need to add this application needs version > 8. Then create a gist on how to install node on Debian and link it there.

Imagine if we have to put the instructions of node installation on all systems.

Comment (azim254) [reply README.md:6]: hehe sounds good

Comment (stannlee) [reply README.md:6]: Hehe have you made the requested changes?

**Comment by azim254** [comment]
okay

**Pull request closed without merging**